app/src/main/java/fr/free/nrw/commons/upload/UploadActivity.java

```diff
@@ -150,13 +154,6 @@ protected void onCreate(Bundle savedInstanceState) {
         compositeDisposable = new CompositeDisposable();
         init();
         nearbyPopupAnswers = new HashMap<>();
-        PermissionUtils.checkPermissionsAndPerformAction(this,
-            this::receiveSharedItems,
-            R.string.storage_permission_title,
-            R.string.write_storage_permission_rationale_for_image_share,
-            PERMISSIONS_STORAGE);
-        //getting the current dpi of the device and if it is less than 320dp i.e. overlapping
```

Comment:
Any particular reason for removing these two lines of comments?

Reply:
reverted.